Library/Homebrew/cmd/update.sh

```diff
@@ -249,6 +252,8 @@ pull() {
     pop_stash_message
   fi
 
+  [[ -n "$HOMEBREW_VERBOSE" ]] && echo
```

Comment:
True. I didn't think about this case, probably because I'm running `brew update` at most every few hours, so there's essentially always something that needs to be fetched. Feel free to leave things as-is or eliminate this line of code completely, whichever you prefer.

The important change is that the output now contains some extra lines that announce what repository `brew update` is currently operating on and that's valuable information. A few missing or superfluous empty lines for readability won't detract from that.